Sources/MountebankSwift/Models/Stub/Response/Behavior.Copy.swift

```diff
@@ -0,0 +1,64 @@
+import Foundation
+
+extension Behavior {
+    public struct Copy: Equatable, Codable {
+        public struct Using: Equatable, Codable {
+            public enum Method: String, Equatable, Codable {
+                case regex
+                case xpath
+                case jsonpath
+            }
+
+            public struct Options: Equatable, Codable {
+                /// Uses a case-insensitive regular expression
+                public let ignoreCase: Bool
+                /// Uses a multiline regular expression
+                public let multiline: Bool
+            }
+
+            /// The method used to select the value(s) from the request.
+            public let method: Method
+            /// The selector used to select the value(s) from the request. For a regex, this would be the pattern,
+            /// and the replacement value will be the entire match. Match groups using parentheses are supported and
+            /// can be replaced using indexed tokens as described in the copy[].into description. xpath and jsonpath
+            /// selectors work on XML and JSON documents. If the request value does not match the selector (including
+            /// through XML or JSON parsing errors), nothing is replaced.
+            public let selector: String
+            /// For xpath selectors, the ns object maps namespace aliases to URLs.
+            public let ns: JSON?
```

Comment:
Perhaps just call this `namespaces`?

Reply:
Oh and also type it `[String: String]` instead of `JSON?

Reply:
Good points, I've updated them in my last commit.